en/docs/develop-api-proxy/policy/secure-client-gateway-communication-with-mtls.md

```
@@ -0,0 +1,93 @@
+# Secure Client-Gateway Communication with Mutual TLS
+
+Mutual TLS (mTLS) adds client-side certificate authentication to ensure only trusted clients can call an API. In WSO2 Bijira you enable mTLS at the API proxy level by configuring the MTLS Authenticator mediation policy to validate client certificates or an intermediate CA chain during the TLS handshake. 
+
+This guide walks through the end-to-end steps on configuring mTLS for an API proxy to consume the API Proxy.
+
+## Prerequisites
+
+1. Set up [ Ballerina 2201.5.5](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.5.5).
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)